schematic/manifest/generator.py

```diff
@@ -1254,7 +1257,7 @@ def get_empty_manifest(self, json_schema_filepath=None, strict=None):
         )
 
         manifest_url = self._create_empty_gs(
```

Comment:
I noticed that `strict` in not in the doc string of function `_create_empty_gs`. Could you add that?

Reply:
Yep!